src/AddTodoForm.jsx

```diff
@@ -0,0 +1,14 @@
+const AddTodoForm = () => {
+    return (
+        <>
+            <form>
+                <label>Title</label>
+                <input id='todoTitle' type='text' />
+                <label htmlFor="todoTitle"></label>
+                <button>Add</button>
```

Comment:
Adding `type="submit"` to the button is a good practice to define its behavior in the form.